### PR TITLE
fix duplicate content issue

### PR DESCRIPTION
**Pull request by pawel-big-lebowski**
Changes: 
 * Prepare reusable partials for transport documentation and circuit breaker
 * Add to Java client subsections: Configuration and Usage Example  with content copied from the main section
 * Add to Spark client subsections: Installation and Configuration with content copied from the main section

### CODE REVIEW

File: static/spec/OpenLineage.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.2
 info:
   title: OpenLineage
-  version: 0.0.1
+  version: 2-0-2
```
Comment (kacpermuda) [inline static/spec/OpenLineage.yml:4]: Is there a particular reason why this value is used?

File: docs/client/java/partials/java_transport.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+import TabItem from '@theme/TabItem';
+
+**Tip:** See current list of [all supported transports](https://github.com/OpenLineage/OpenLineage/tree/main/client/java/src/main/java/io/openlineage/client/transports).
+
+### [HTTP](https://github.com/OpenLineage/OpenLineage/tree/main/client/java/src/main/java/io/openlineage/client/transports/HttpTransport.java)
+
+Allows sending events to HTTP endpoint (with optional authorization headers).
+
+<Tabs groupId="spark">
+<TabItem value="OpenLineageYaml Config" label="OpenLineageYaml Config">
```
Comment (kacpermuda) [inline docs/client/java/partials/java_transport.md:11]:
```suggestion
<TabItem value="Yaml Config" label="Yaml Config">
```
I would go with this everywhere, but have no strong arguments - just preference

File: docs/client/java/partials/java_transport.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+```yaml
+transport:
+  type: http
+  url: http://localhost:5000
+```
+
+*YAML Configuration* (with authorization)
```
Comment (kacpermuda) [inline docs/client/java/partials/java_transport.md:19]:
```suggestion
*with authorization*
```

File: docs/client/java/partials/java_transport.md [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+
+
+### [Console](https://github.com/OpenLineage/OpenLineage/tree/main/client/java/src/main/java/io/openlineage/client/transports/ConsoleTransport.java)
+
+This straightforward transport emits OpenLineage events directly to the console through a logger.
+Be cautious when using the DEBUG log level, as it might result in double-logging due to the `OpenLineageClient` also logging.
+No additional configuration is required.
+
+<Tabs groupId="integrations">
+<TabItem value="yaml" label="OpenLineageYaml Config">
```
Comment (kacpermuda) [inline docs/client/java/partials/java_transport.md:215]: Is there no spark / flink config here?

File: docs/client/java/mqz_job_complete.png
Comment (kacpermuda) [inline docs/client/java/mqz_job_complete.png:1]: Is this the right place to store images? If there is no dedicated directory, maybe we should create one on the repository level?

File: docs/integrations/spark/installation.md [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+
+:::info
+This approach does not demonstrate how to configure the `OpenLineageSparkListener`.
+Please refer to the [Configuration](./configuration.md) section.
+:::
+
+For Maven, add the following to your `pom.xml`:
+
+<Tabs groupId="spark">
+<TabItem value="after-1.8.0" label="After 1.8.0">
```
Comment (kacpermuda) [inline docs/integrations/spark/installation.md:33]: Not sure if the tabs work properly in this file, at least in the [preview](https://deploy-preview-290--mellifluous-eclair-326dcd.netlify.app/docs/integrations/spark/installation) here they did not render at all

File: docs/integrations/spark/configuration.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+a `SparkSession`.
+
+:::warning
+The setting `config("spark.extraListeners", "io.openlineage.spark.agent.OpenLineageSparkListener")` is
+**extremely important**. Without it, the OpenLineage Spark integration will not be invoked, rendering
+the integration ineffective.
+:::
+
+<Tabs groupId="spark-app-conf">
+<TabItem value="scala" label="Scala">
```
Comment (kacpermuda) [inline docs/integrations/spark/configuration.md:27]: Here i got the same, no tabs are rendered.

File: docs/integrations/spark/spark.md
Comment (kacpermuda) [inline docs/integrations/spark/spark.md:1]: I think this is no longer needed here?
```
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';
```

**Review by kacpermuda — COMMENTED** [review]
Amazing work 🚀 Leaving some small comments to make sure everything works as expected.

File: docs/integrations/spark/installation.md (hunk 5 above)
Comment (kacpermuda) [reply docs/integrations/spark/installation.md:33]: On localhost it looks the same, fyi. F.e. these are the tabs for scala and python: 
image

**Comment by kacpermuda** [comment]
Also, i was trying to find a simple way to "pre-choose" the displayed tab at the import location, so that we can by default show Spark config tab in Spark docs, Flink config tabs in Flink etc. I did not find en easy way to do that, but maybe there is one ? That would be a nice cherry on top 😄

**Comment by pawel-big-lebowski** [comment]
> Also, i was trying to find a simple way to "pre-choose" the displayed tab at the import location, so that we can by default show Spark config tab in Spark docs, Flink config tabs in Flink etc. I did not find en easy way to do that, but maybe there is one ? That would be a nice cherry on top 😄

Was also looking into this, but couldn't find it ;-)

File: docs/integrations/spark/installation.md (hunk 5 above)
Comment (pawel-big-lebowski) [reply docs/integrations/spark/installation.md:33]: hmyy. perhaps I missed an import -> will look into this.

File: docs/client/java/mqz_job_complete.png
Comment (pawel-big-lebowski) [reply docs/client/java/mqz_job_complete.png:1]: don't know. I would say what you propose is unrelated to my PR but perhaps worth further improvement

File: docs/client/java/partials/java_transport.md (hunk 4 above)
Comment (pawel-big-lebowski) [reply docs/client/java/partials/java_transport.md:215]: Yeah. `transport.type` is sufficient and this is documented elsewhere. I thought empty tab would be better than no tab.

File: docs/integrations/spark/configuration.md (hunk 6 above)
Comment (pawel-big-lebowski) [reply docs/integrations/spark/configuration.md:27]: missing import. Thanks for spotting this.

File: static/spec/OpenLineage.yml (hunk 1 above)
Comment (pawel-big-lebowski) [reply static/spec/OpenLineage.yml:4]: this shouldn't be within the PR - no particular reason.

**Review by kacpermuda — APPROVED** [review]
🎉

**Pull request merged**